crates/stc_ts_file_analyzer/tests/pass/exprs/typeGuards/typeGuardsWithInstanceOfByConstructorSignature/2.swc-stderr

```diff
@@ -0,0 +1,92 @@
+
+  x Type
+    ,-[$DIR/tests/pass/exprs/typeGuards/typeGuardsWithInstanceOfByConstructorSignature/2.ts:11:1]
+ 11 | if (obj4 instanceof B) {
+    :     ^^^^
+    `----
+
+Error: 
+  > any
+
+  x Type
+    ,-[$DIR/tests/pass/exprs/typeGuards/typeGuardsWithInstanceOfByConstructorSignature/2.ts:11:1]
+ 11 | if (obj4 instanceof B) {
+    :                     ^
+    `----
+
+Error: 
+  > BConstructor
+
+  x Type
+    ,-[$DIR/tests/pass/exprs/typeGuards/typeGuardsWithInstanceOfByConstructorSignature/2.ts:11:1]
+ 11 | if (obj4 instanceof B) {
+    :     ^^^^^^^^^^^^^^^^^
+    `----
+
+Error: 
+  > boolean
+
+  x Type
+    ,-[$DIR/tests/pass/exprs/typeGuards/typeGuardsWithInstanceOfByConstructorSignature/2.ts:12:3]
+ 12 | obj4; // B<any>
+    : ^^^^
+    `----
+
+Error: 
+  > B
+
+  x Type
+    ,-[$DIR/tests/pass/exprs/typeGuards/typeGuardsWithInstanceOfByConstructorSignature/2.ts:13:3]
+ 13 | obj4.foo = "str";
+    : ^^^^
+    `----
+
+Error: 
+  > B
+
+  x Type
+    ,-[$DIR/tests/pass/exprs/typeGuards/typeGuardsWithInstanceOfByConstructorSignature/2.ts:13:3]
+ 13 | obj4.foo = "str";
+    : ^^^^^^^^
+    `----
+
+Error: 
+  > {
```

Comment:
Same here

Reply:
wrong type infer is problem arises because of generic, I don't know that.